src/GMBuddy.Games/Micro20/InputModels/CharacterModification.cs

```diff
@@ -29,7 +31,38 @@ public class CharacterModification
         [Range(3, 18)]
         public int? Mind { get; set; }
 
-        [Range(1, 100)]
+        [Range(0,100)] //assuming a level 20 cap
+        public int? Physical { get; set; }
+
+        [Range(0,100)] //assuming a level 20 cap
+        public int? Subterfuge { get; set; }
+
+        [Range(0,100)] //assuming a level 20 cap
+        public int? Knowledge { get; set; }
+
+        [Range(0,100)] //assuming a level 20 cap
+        public int? Communication { get; set; }
+
+        [Range(1, int.MaxValue)]
         public int? Level { get; set; }
+
+        [Range(1, int.MaxValue)]
+        public int? Experience { get; set;} //when modifying a character, if experience is ever equal or greater than 10xlevel, player advances a level, then experience resets (extra exp rolls over)
+
+        internal ICollection<Item> Items { get; set; }
```

Comment:
Why did you mark these all as internal... When I mentioned Internal before, I meant the class, and that was only if you had a corresponding Input/Output model for it. if youre just going to use the raw database model, dont mark it as internal. Internal also makes it untestable in our current setup...